### DISCUSSION TITLE
kunDb  没有专属的驱动  c# 目前只能用 odbc,  FreeSql 支持odbc吗 ？

### DISCUSSION THREAD

**Discussion by lishi1841** [dotnetcore/FreeSql#1070] [Q&A]
kunDb  没有专属的驱动  c# 目前只能用 odbc,  FreeSql 支持odbc吗 ？

**Comment by 2881099** [comment]
FreeSql.Provider.Odbc.dll 就可以

或者用 FreeSql.Provider.Custom.dll 自定义 

https://github.com/dotnetcore/FreeSql/tree/master/Providers/FreeSql.Provider.Custom
